solr/contrib/ltr/src/java/org/apache/solr/ltr/util/CommonLTRParams.java

```diff
@@ -112,4 +112,6 @@
 
   // Name of key used to store the original score of a doc
   public static final String ORIGINAL_DOC_SCORE = "ORIGINAL_DOC_SCORE";
+  public static final String FEATURE_REQUEST = "[features]";
```

Comment:
The transformer name is dynamic based on what you name it in the solrconfig.xml.  It isn't hard coded to [features] or [fv].  These variables wont help you.